tests/tests.js

```diff
@@ -4,42 +4,137 @@ const employee = require('./employee')
+  })
+
+  describe('calculateVolLifePrice', () => {
+    it('returns the toal price for a voluntary life product for a single employee without employer contribution', () => {
+      const selectedOptions = {
+        familyMembersToCover: ['ee'],
+        coverageLevel: [{ role: 'ee', coverage: 125000 }],
+      }
+      const price = pricing.calculateVolLifePrice(products.voluntaryLife, selectedOptions)
+
+      expect(price).to.equal(43.75)
+    })
+    it('returns the price for a voluntary life product for an employee with a spouse without employer contribution', () => {
+      const selectedOptions = {
+        familyMembersToCover: ['ee', 'sp'],
+        coverageLevel: [
+          { role: 'ee', coverage: 200000 },
+          { role: 'sp', coverage: 75000 },
+        ],
+      }
+      const price = pricing.calculateVolLifePrice(products.voluntaryLife, selectedOptions)
+
+      expect(price).to.equal(79)
+    })
   })
 
-  it('returns the price for a voluntary life product for an employee with a spouse', () => {
-    const selectedOptions = {
-      familyMembersToCover: ['ee', 'sp'],
-      coverageLevel: [
-        { role: 'ee', coverage: 200000 },
+  describe('calculateVolLifePricePerRole', () => {
+    it('return the VolLifePrice product for each role of employee', () => {
+      const role = 'ee'
+      const coverageLevel = [{ role: 'ee', coverage: 200000 },]
+
+      const price = pricing.calculateVolLifePricePerRole(role, coverageLevel, products.voluntaryLife.costs)
+
+      expect(price).to.equal(70)
+    })
+    it('return the VolLifePrice product for each role of employee', () => {
+      const role = 'sp'
+      const coverageLevel = [
         { role: 'sp', coverage: 75000 },
-      ],
-    }
-    const price = pricing.calculateProductPrice(products.voluntaryLife, employee, selectedOptions)
+      ]
+
+      const price = pricing.calculateVolLifePricePerRole(role, coverageLevel, products.voluntaryLife.costs)
 
-    expect(price).to.equal(71.09)
+      expect(price).to.equal(9)
+    })
   })
 
-  it('returns the price for a disability product for an employee', () => {
-    const selectedOptions = {
-      familyMembersToCover: ['ee']
-    }
-    const price = pricing.calculateProductPrice(products.longTermDisability, employee, selectedOptions)
+  describe('getEmployerContribution', () => {
+    it('returns the employer contribution for voluntary life product', () => {
+      const price = 44.00
+      const product = products.voluntaryLife
+      employerContribution = pricing.getEmployerContribution(product.employerContribution, price)
+      expect(employerContribution).to.equal(4.4)
+    })
 
-    expect(price).to.equal(22.04)
+    it('returns the employer contribution for LTD product ', () => {
+      const price = 44.00
+      const product = products.longTermDisability
+      employerContribution = pricing.getEmployerContribution(product.employerContribution, price)
+      expect(employerContribution).to.equal(10)
+    })
```

Comment:
While these tests cover the two cases you are hiding the details of how they work by having your tests be for each product. The function has two paths, one for percentage based and one for dollar based. That's what these tests are actually testing. It just so happens that LTD and VolLife are of different contribution types.